poetry/console/commands/show.py

```diff
@@ -144,12 +145,31 @@ def handle(self):
                 if not self.option("outdated") or update_status != "up-to-date":
                     name_length = max(name_length, current_length)
                     version_length = max(
-                        version_length, len(locked.full_pretty_version)
+                        version_length,
+                        len(
+                            get_package_version_display_string(
```

Comment:
Maybe this can go in a variable?

Reply:
How so? We are not using it in a lot of places and they are used for different packages (`locked`/`latest`).

Reply:
Mostly for readability. I am okay with either.

Reply:
I think we will leave this as-is for now. It may change anyway in `1.2` with the refactoring that will likely happen.